src/components/Game.tsx

```diff
@@ -84,6 +85,7 @@ export const Game: FC = (): JSX.Element => {
   };
 
   useHandleButtons(updateState);
+  const [onTouchStart, onTouchEnd] = useHandleTouches((dir) => updateState(dir));
```

Comment:
This line can be simplified to `const [onTouchStart, onTouchEnd] = useHandleTouches(updateState);`